pkg/controllers/reconciler.go

```diff
@@ -221,6 +320,46 @@ func (r *reconciler) updateBookKeeper(ctx context.Context) error {
 	return nil
 }
 
+func (r *reconciler) createClusterRoleBinding(ctx context.Context, dp client.Object, log logr.Logger) (ctrl.Result, error) {
+	rb := r.controller.NewClusterRoleBinding(dp)
+	if rb == nil {
+		// most controllers don't need role bindings
+		return ctrl.Result{}, nil
+	}
+
+	if err := ctrl.SetControllerReference(dp.(metav1.Object), rb, r.scheme); err != nil {
+		log.Error(err, "unable to set controller reference")
+		return ctrl.Result{}, err
+	}
+
+	if err := r.Create(ctx, rb); err != nil {
+		log.Error(err, "unable to create ClusterRoleBinding")
+		return ctrl.Result{}, err
+	}
+
+	return ctrl.Result{}, nil
+}
+
+func (r *reconciler) createServiceAccount(ctx context.Context, dp client.Object, log logr.Logger) (ctrl.Result, error) {
+	sa := r.controller.NewServiceAccount(dp)
+	if sa == nil {
+		// most controllers don't need service accounts
```

Comment:
I thought all controllers needed service accounts. They just don't create them programmatically. IIRC they used to run under service accounts created for them via YAML configs, but at some point we switched to the default service accounts created automatically by K8s. Can't the GPU manager use the default service account too?

Reply:
I could rephrase the comment with something descriptive like "special service accounts". The default service account would need quite a bit of beefing up in terms of rbac permissions. I had this idea here that I would have a dedicated service account just because giving the rbac permissions to everybody who uses the "default" isn't necessarily the most secure way of doing it.

Reply:
Oh, I see this for the operator. These changes can go to a separate PR to keep this one reviewable.